controls/SV-257840.rb

```diff
@@ -25,4 +25,9 @@
   tag 'documentable'
   tag cci: ['CCI-000366']
   tag nist: ['CM-6 b']
+  tag 'host', 'container'
```

Comment:
nitpick: should be on separate lines
```suggestion
  tag 'host'
  tag 'container'
```